server/jobs/onboarding/email.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-param-reassign */
+import { developerEmailTransport, opsEmailTransport } from '../../modules/email/emailModule';
+/**
+ * @desc Automates developer onboarding on email
+ *
+ * @param {array} placement Placement record whose developer is to be offboarded
+ * @param {object} automationResult Result of automation job
+ * @returns {void}
+ */
+const emailOnboarding = async (placement, automationResult) => {
+  const { fellow } = placement;
+  const { client_name: partnerName } = placement;
+  const { client_location: partnerLocation } = placement;
+  const { start_date: startDate } = placement;
+  try {
+    await developerEmailTransport(fellow.email, fellow.name, partnerName);
+    // eslint-disable-next-line max-len
+    await opsEmailTransport(fellow.name, partnerName, fellow.email, fellow.location, partnerLocation, startDate);
+    automationResult.slackOnBoarding = 'success';
+  } catch (error) {
+    automationResult.slackOnBoarding = 'failure';
```

Comment:
I guess you forgot to change something here, this isn't `slackOnBoarding`.

Reply:
I have resolved it